pkgs/top-level/all-packages.nix

```diff
@@ -16474,6 +16474,7 @@ in
     python = python27;
     qmake = qt59.qmake;
     qtwebengine = qt59.qtwebengine;
+    qtxmlpatterns=pkgs.qt59.qtxmlpatterns;
```

Comment:
I guess `csound-qt` should just use `libsForQt59.callPackage` instead of these three lines.

Reply:
I'll try

Reply:
Works